github-orgs/kubeflow/org.yaml

```diff
@@ -961,6 +952,35 @@ orgs:
             privacy: closed
             repos:
               website: write
+          wg-automl-leads:
```

Comment:
I think we can drop "wg"; in the related issue. 
wg was supposed to be a placeholder for the actual work group.

Reply:
@jlewi Done.